docs/guides/deployment/digitalocean.rst

```diff
@@ -0,0 +1,180 @@
+
+To change the password run the following. You will find you droplet ip address
+on digitalocean_. The EdgeDB cli installer can be found here_.
+
+.. _digitalocean: https://cloud.digitalocean.com/droplets?
+.. _here: edgedb-install_
+
+.. code-block:: bash
+
+   $ IP=<your-droplet-ip>
+   $ read -sp "Password: " PASSWORD
+   $ printf edgedbpassword | edgedb query \
+         --host $IP \
+         --password-from-stdin \
+         --tls-security insecure \
+         "alter role edgedb set password := '${PASSWORD}'"
+
+Optionally create a local link to your new EdgeDB server.
+
+.. code-block:: bash
+
+   $ printf $PASSWORD | edgedb instance link \
+         --host $IP \
+         --trust-tls-cert \
+         --password-from-stdin \
+         --non-interactive \
+         digitalocean
+
+
+.. _ref_guide_deployment_digitalocean_managed:
+
+Deploy with Managed PostgreSQL
+++++++++++++++++++++++++++++++
+
+Prerequisites
+=============
+
+* ``edgedb`` CLI (`install <edgedb-install_>`_)
+* DigitalOcean account
+* ``doctl`` CLI (`install <doclt-install_>`_)
+* ``jq`` (`install <jq_>`_)
+
+.. _edgedb-install: https://www.edgedb.com/install
+.. _doclt-install: https://docs.digitalocean.com/reference/doctl/how-to/install
+.. _jq: https://stedolan.github.io/jq/
+
+
+Create a managed PostgreSQL instance
+====================================
+
+If you already have a PostgreSQL instance you can skip this step.
+
+.. code-block:: bash
+
+   $ DSN="$( \
+         doctl databases create edgedb-postgres \
+             --engine pg \
+             --version 12 \
+             --size db-s-1vcpu-1gb \
+             --num-nodes 1 \
+             --region sfo3 \
+             --output json \
+         | jq -r '.[0].connection.uri' )"
+
+
+Provision a droplet
+===================
+
+Replace ``$SSH_KEY_IDS`` with the ids for the ssh keys you want to ssh into the
+new droplet with. Separate multiple values with a comma. You can list your
+keys with ``doctl compute ssh-key list``.  If you don't have any ssh keys in
+your DigitalOcean account you can follow `this guide <upload-ssh-keys_>`_ to do
+that now.
+
+.. _upload-ssh-keys:
+   https://docs.digitalocean.com/products/droplets
+   /how-to/add-ssh-keys/to-account/
+
+.. code-block:: bash
+
+   # XXX: we don't have a published edgedb image yet
+   $ IP="$( \
+         doctl compute droplet create edgedb \
+             --image edgedb-1-rc3 \
+             --region sfo3 \
+             --size s-2vcpu-4gb \
+             --ssh-keys $SSH_KEY_IDS \
+             --format PublicIPv4 \
+             --no-header \
+             --wait )"
+
+Configure the backend postgres DSN. To simplify the initial deployment, let's
+instruct EdgeDB to run in insecure mode (with password authentication off and
+an autogenerated TLS certificate). We will secure the instance once things are
+up and running.
+
+.. code-block:: bash
+
+   $ ssh root@$IP "
+   cat << EOF > /etc/edgedb/env
+   EDGEDB_SERVER_BACKEND_DSN=${DSN}
```

Comment:
It looks like the docs generation requires a `\` at the end of a line for it to treat the next line as part of the same command.
![image](https://user-images.githubusercontent.com/8278635/151076160-a505a639-97a1-4be6-a85e-b047185fcb53.png)
This snippet can't be copied using the copy button when rendered to HTML. It only copies the first line :(

Reply:
cc @elprans

Reply:
Rewrite it as

```
printf "..." | ssh root@$IP -T "cat > /etc/edgedb/env"
```

Reply:
I still can't get the string to fit on one line so it has the same problem.
![image](https://user-images.githubusercontent.com/8278635/151215591-c9b72a68-a095-486f-a131-5fb7428bd948.png)

I could do it with two commands but this seems strange in a guide.
![image](https://user-images.githubusercontent.com/8278635/151215789-811bc75a-b9c4-49a3-ab80-f9fc7d48fab8.png)

Reply:
Try

```
printf "EDGEDB_SERVER_BACKEND_DSN=${DSN}\n\
EDGEDB_SERVER_SECURITY=insecure_dev_mode\n" | ...
```

Reply:
There has to be a space before the `\` for it to count as a line continuation by the doc generation system but this one does work. The placement of `\n`s is a little strange but its not too bad.
![image](https://user-images.githubusercontent.com/8278635/151219270-fc6da64c-7ed3-4a72-a8bc-feb34e4001a0.png)